lib/filters.js

```diff
@@ -71,8 +74,8 @@ export const refFilter = (data, { domain, domainOnly, raw }) => {
 
       if (protocol.startsWith('http')) {
         const path = removeTrailingSlash(pathname);
-        const ref = searchParams.get('ref');
-        const query = ref ? `?ref=${ref}` : '';
+        const referrer = searchParams.get('ref');
```

Comment:
The .get is still calling `ref`, is this intended?

Reply:
Ah, good catch, will update